src/AltinnCore/Common/Services/Interfaces/IRepository.cs

```diff
@@ -451,5 +451,13 @@ public interface IRepository : IServicePackageRepository
         /// <param name="applicationInformation">the application information to be updated</param>
         /// <returns>true if the information is updated successfully</returns>
         bool UpdateServiceInformationInApplicationMetadata(string org, string applicationId, ServiceConfiguration applicationInformation);
+
+        /// <summary>
+        /// Returns the applicaiton metadata for an application
+        /// </summary>
+        /// <param name="applicationOwnerId">the applicatio owner</param>
```

Comment:
Are we still using application owner id? I thought that the discussion ended with us using owner? @dskogan @eimik

Reply:
@hannekot it was agreed to 'org', see #1218 , I forgot to update it to org